### PR TITLE
[UR][L0] Make urPlatformGetBackendOption return -ze-opt-level=2 for -O1 and -O2

### PR DESCRIPTION
**Pull request by sarnex**
https://github.com/oneapi-src/unified-runtime/pull/1129

### CODE REVIEW

**Comment by kbenzie** [comment]
We're looking to merge the UR changes for this soon, opening this up for review so that we have the necessary approvals for when we do merge.

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -56,12 +56,12 @@ endif()
 if(SYCL_PI_UR_USE_FETCH_CONTENT)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/sarnex/unified-runtime.git")
```
Comment (maarquitos14) [inline sycl/plugins/unified_runtime/CMakeLists.txt:59]: I guess you forgot to change this back?

**Review by kbenzie — APPROVED** [review]

File: sycl/plugins/unified_runtime/CMakeLists.txt (hunk 1 above)
Comment (kbenzie) [reply sycl/plugins/unified_runtime/CMakeLists.txt]: This is updated now.

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers please merge

**Pull request merged**